services/web/client/source/class/osparc/desktop/preferences/pages/ConfirmationsPage.js

```diff
@@ -103,11 +103,41 @@ qx.Class.define("osparc.desktop.preferences.pages.ConfirmationsPage", {
       const cbConfirmDeleteStudy = new qx.ui.form.CheckBox(this.tr("Delete a Study"));
       preferencesSettings.bind("confirmDeleteStudy", cbConfirmDeleteStudy, "value");
       cbConfirmDeleteStudy.bind("value", preferencesSettings, "confirmDeleteStudy");
+      cbConfirmDeleteStudy.addListener("changeValue", e => {
+        if (!e.getData()) {
+          const msg = this.tr("\
+            Are you sure you want to disable this confirmation?\
```

Comment:
this is probably sufficient just as a warning e.g. "Warning: deleting a study cannot be undone" with an "ok" button, However if it would take longer than 5 min to do that, don't worry about it

Reply:
Done!